src/main/java/com/esri/samples/map/create_and_save_map/README.md

```diff
@@ -0,0 +1,39 @@
+<h1>Create and Save Map</h1>
+
+<p>How to create and save a map to your own portal.</p>
+
+<p><img src="CreateAndSaveMap.png"/></p>
```

Comment:
I've sent you credentials to use. As for the first comment, the sample is really more about saving than oauth, so I'd rather not overly complicate the image.